### PR TITLE
single isd format + mro test conversion

### PR DESCRIPTION
**Pull request by Kelvinrr**

### CODE REVIEW

File: ale/drivers/__init__.py [hunk 1]
```diff
@@ -32,7 +33,8 @@
 __driver_modules__ = [importlib.import_module('.'+m, package='ale.drivers') for m in __all__]
 
 __formatters__ = {'usgscsm': to_usgscsm,
-                  'isis': to_isis}
+                  'isis': to_isis,
+                  'ale' : to_isd}
```
Comment (Kelvinrr) [inline ale/drivers/__init__.py]: Seems like a way to get functionality for instruments without full support for both ISIS and CSM formatters is to simply make the singular ISD a new formatter and slowly start phasing out the other two.

Comment (jessemapel) [inline ale/drivers/__init__.py]: It looks like to_isd is missing from the PR.

**Comment by Kelvinrr** [comment]
Presumably, if this looks good, this is mergeable and we can place other PRs as we move through instruments.

**Comment by jessemapel** [comment]
For the sake of getting to moving to the new format, I'm okay with merging this and then adding the test in another PR.

**Pull request merged**